stock_release_channel_process_end_time/models/stock_release_channel.py

```diff
@@ -100,3 +94,17 @@ def _field_picking_domains(self):
                 new_domain.append(criteria)
         res["count_picking_release_ready"] = new_domain
         return res
+
+    def _get_expected_date(self):
+        """Return the new date to set on move chain"""
```

Comment:
Better to use comment instead of docstrings when overriding methods (this overrides original docstrings in IDE and if the method is triggered by a job, the description of the job = the last docstring applied on the method, which is often misleading).